### PR TITLE
Remove redirecting links

### PR DESCRIPTION
**Pull request by teallarson**
## Change description

- Removes links to redirecting pages
- Removes `/solutions/marketers`
- Tests against the `next.config` for any internal redirects being linked to.  External links such as using `/roadmap` to link to  our github roadmap have been ignored because they were not listed as negatively affecting SEO

unrelated cleanup:
- Adds Models to the core concepts list.  The blurb was there, but not a link to it at the top.

## Related issues

Does this PR fix a Github Issue?

no

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

>

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: components/footer.tsx [hunk 1]
```diff
@@ -105,14 +105,6 @@ export default function Footer() {
...
                   <strong>Use Cases</strong>
                   <br />
-                  <Link href="/solutions/engineers">
-                    <a>Grouparoo for Engineers</a>
-                  </Link>
-                  <br />
-                  <Link href="/solutions/marketers">
-                    <a>Grouparoo for Marketers</a>
-                  </Link>
-                  <br />
```
Comment (evantahler) [inline components/footer.tsx:115]: 👋

**Review by evantahler — APPROVED** [review]
Approved as-is, but if you wanted, I think there is room for some structural improvements:

* Should these have been relative links?  eg: rather than `http://www.grouparoo.com/legal/privacy` we link to `/legal/privacy?`
* Should we have a test to ensure that we don't forget and link to `/jobs` in the future?

**Comment by teallarson** [comment]
@evantahler maybe a global test that we don't link to a URL that redirects would be useful for that and other cases?  We should be able to read them from `next.config.js` and compare somehow?

File: __tests__/integration/html.ts [hunk 2]
```diff
@@ -58,6 +59,21 @@ describe("sitemap integration", () => {
...
+    const redirectPaths = redirects.map((redirect) => {
+      if (redirect.destination.startsWith("/")) return redirect.source;
+    });
+
+    for (const i in redirectPaths) {
+      if (link === redirectPaths[i]) {
+        return true;
+      }
+    }
+  };
```
Comment (evantahler) [inline __tests__/integration/html.ts:75]: Nice!

**Review by evantahler — APPROVED** [review]

**Pull request merged**